### PR TITLE
Minor updates

### PR DESCRIPTION
**Pull request by kpmmmurphy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -122,16 +126,16 @@ Alerter.create(this)
 ```java
  Alerter.create(ExampleActivity.this)
         .setTitle("Alert Title")
-        .setOnShownListener(new OnAlertShownListener() {
+        .setOnShownListener(new OnShowAlertListener() {
```
Comment (otormaigh) [inline README.md]: Rename these to match the method names `setOnShowListener`

File: alerter/src/main/java/com/tapadoo/alerter/OnHideAlertListener.java [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+ *
+ * @author edgeorge
+ * @since 20/02/2017.
+ */
+
+public interface OnHideAlertListener {
+    /**
+     * Called when the Alert is hidden
+     */
+    void onHide();
```
Comment (otormaigh) [inline alerter/src/main/java/com/tapadoo/alerter/OnHideAlertListener.java]: Could add a onDismiss listener too. But wonder would it ever be useful?

File: README.md (hunk 1 above)
Comment (kpmmmurphy) [inline README.md]: Good catch!

File: README.md [hunk 3]
```diff
@@ -10,11 +10,15 @@ complexity of your layouts.
 A customisable Alert view is dynamically added to the Decor View of the Window, overlaying
 all content.
 
+[ ![Download](https://api.bintray.com/packages/tapadoo/maven/alerter/images/download.svg) ](https://bintray.com/tapadoo/maven/alerter/_latestVersion)
```
Comment (otormaigh) [inline README.md]: Should move these badges to the top of the README. Are you able to inline them too?

File: alerter/src/main/java/com/tapadoo/alerter/OnHideAlertListener.java (hunk 2 above)
Comment (kpmmmurphy) [inline alerter/src/main/java/com/tapadoo/alerter/OnHideAlertListener.java]: Good point - I think they'd be functionally the same thing, as this is executed when the Alert view is removed form the parent, either after the show duration has expired, or if the user touches the alert to dismiss it.

File: README.md (hunk 3 above)
Comment (kpmmmurphy) [inline README.md]: I'll move 'em on up!

Comment (kpmmmurphy) [inline README.md]: Not sure about inlining!

**Pull request merged**